### ISSUE TITLE
InlayHint: Remaining tasks

### ISSUE THREAD

**Issue by dantleech** [phpactor/phpactor#2139]
- [ ] Show inlay param hint for function
- [ ] Show inlay param hint for class instantation

**Comment by lumnn** [comment]
It looks like in sublime there is a chance to show extra data on hover

[Screencast from 2023-03-16 15-42-23.webm](https://user-images.githubusercontent.com/2505114/225653151-1f7fc184-a3b7-4113-add4-ad306418a8ee.webm)

**Comment by dantleech** [comment]
yes -- currently i set that to the type, but it's perhaps redundant, not sure what to put there, maybe nothing.

**Comment by lumnn** [comment]
I just saw <missing> and I thought at least the param type could be shown there, but it is actually - it's just it takes only PHP types not the phpdoc one.
However, types as an inline hint (not the hover info) shows phpdoc ones.
